src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Figure implements FigureDraw {

```

Comment:
According to the checklist, you should not begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class implementation.